src/aclib/ac_utils/ac_utils.H

```diff
@@ -156,6 +152,149 @@ void sigusr2_handler(int signal)
 #endif //AC_COMPSIM
 
 
+//Read model options before application
+void ac_init_opts( int ac, char* av[]){
```

Comment:
Since you are moving this code and git blame is going to point to you anyway, you might as well clang-format this entire thing.